gertie/Sources/Gertie/CGImage+isNearlyIdentical.swift

```diff
@@ -0,0 +1,31 @@
+import Accelerate
+
+public extension CGImage {
+  func isNearlyIdenticalTo(_ other: CGImage?) -> Bool {
+
+    guard let other,
```

Comment:
wait one sec, just saw that you removed the `width == other.width` and height tests.  on iOS this is superfluous, but i don't think on macos, because it's common to plugin in and remove external monitors, so the screen size is not permanently fixed. i suppose the other check will catch it, but i sort of like keeping it here.